src/AdafruitIO.cpp

```diff
@@ -241,7 +253,7 @@ aio_status_t AdafruitIO::mqttStatus()
 
   if(_mqtt->connected())
     return AIO_CONNECTED;
-
+  
```

Comment:
remove newline here

Reply:
removed spaces so it should show unmodified, with one blank line